.github/ISSUE_TEMPLATE/submit-your-project--.md

```diff
@@ -21,6 +21,8 @@ Please fill in the following info and submit the Issue.
 **Organizers**:
 <!--Your name and GitHub login, possibly more than 1 lead-->
 
+**Mattermost handle**:
+
```

Comment:
We have a chat channel at the bottom. If you think it is not clear enough we can change that one.

Reply:
yes, but I thought it asks for channel link. I was thinking maybe we should ask for handle also so as to add all teammates in our general channel for announcements etc.